src/cgame/cg_rocket_draw.cpp

```diff
@@ -917,6 +917,36 @@ class SpeedGraphElement : public HudElement
 	Color::Color backColor;
 };
 
+class PositionElement : public TextHudElement
+{
+public:
+	PositionElement( const Rml::Core::String& tag ) :
+			TextHudElement( tag, ELEMENT_GAME, true )
+	{
+	}
+
+	void DoOnRender() override
+	{
+		if ( !cg_drawPosition.Get() )
+		{
+			SetText( "" );
+		}
+		else
+		{
+			if ( cg.predictedPlayerState.clientNum == cg.clientNum )
+			{
+				// Add text to be configured via CSS
+				vec3_t origin;
+				VectorCopy( cg.predictedPlayerState.origin, origin );
+
+				// HACK: Put extra spaces to separate the children because setting them to block makes them disappear.
+				// TODO: Figure out why setting these two elements to block makes them disappear.
```

Comment:
Please fix or remove todos if you're satisfied with the current result.

Reply:
I actually duplicated code, and merging this would increase tech debt. I don't think it's a good idea.

Reply:
Fun fact, it was actually no longer useful, code works nicely *without* those. I also removed the useless copy...